cfme/tests/services/test_dialog_regex_validation_in_catalog.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.services.service_catalogs import ServiceCatalogs
+from cfme.utils.appliance.implementations.ui import navigate_to
+from wait_for import wait_for
+
+pytestmark = [
+    test_requirements.service,
+    pytest.mark.tier(2)
+]
+
+
+@pytest.fixture(scope="function")
+def dialog_cat_item(appliance, request, catalog):
+    service_dialog = appliance.collections.service_dialogs
+    dialog = "dialog_{}".format(fauxfactory.gen_alphanumeric())
+    ele_name = fauxfactory.gen_alphanumeric()
+    element_data = {
+        'element_information': {
+            'ele_label': "ele_{}".format(fauxfactory.gen_alphanumeric()),
+            'ele_name': ele_name,
+            'ele_desc': fauxfactory.gen_alphanumeric(),
+            'choose_type': "Text Box"
+        },
+        'options': {
+            'validation_switch': True,
+            'validation': "^([a-z0-9]+_*)*[a-z0-9]+$"
+            }
+        }
+    if appliance.version < '5.10':
+        element_data["options"].pop("validation_switch", None)
+    sd = service_dialog.create(label=dialog, description="my dialog")
+    tab = sd.tabs.create(tab_label="tab_{}".format(fauxfactory.gen_alphanumeric()),
+                         tab_desc="my tab desc")
+    box = tab.boxes.create(box_label="box_{}".format(fauxfactory.gen_alphanumeric()),
+                           box_desc="my box desc")
+    box.elements.create(element_data=[element_data])
+    request.addfinalizer(sd.delete_if_exists)
+
+    item_name = fauxfactory.gen_alphanumeric()
+    catalog_item = appliance.collections.catalog_items.create(
+        appliance.collections.catalog_items.GENERIC,
+        name=item_name,
+        description="my catalog",
```

Comment:
@niyazRedhat 
According to me, instead of creating extra variable ```item_name``` just use ```name=fauxfactory.gen_alphanumeric()```. Although this variable is used once.